### PR TITLE
Improve Spec Builder

### PR DESCRIPTION
**Pull request by manuyavuz**
`weak_frameworks` attribute was not being added to generated spec. Also there was a problem with multi-line string attributes like `description`

Could you check @neonichu?

### CODE REVIEW

File: lib/cocoapods-packager/spec_builder.rb [hunk 1]
```diff
@@ -59,12 +49,17 @@ def spec_header
...
 
       %w(name version summary license authors homepage description social_media_url
-         docset_url documentation_url screenshots frameworks libraries requires_arc
+         docset_url documentation_url screenshots frameworks weak_frameworks libraries requires_arc
          deployment_target xcconfig).each do |attribute|
         value = @spec.attributes_hash[attribute]
         next if value.nil?
-
-        value = "'#{value}'" if value.class == String
+        if value.class == String
```
Comment (segiddins) [inline lib/cocoapods-packager/spec_builder.rb]: why is this necessary?

Comment (manuyavuz) [inline lib/cocoapods-packager/spec_builder.rb]: Yeah, it could, but then we lose the readability inside the generated podspec.

**Comment by manuyavuz** [comment]
Is there anything needed for this to be merged?

**Comment by neonichu** [comment]
:ship:
